lib/HTFeed/Stage/Unpack.pm

```diff
@@ -87,10 +92,15 @@ sub _extract_file {
         return;
     }
 
+    # We don't use $self here, so we have to get job_metrics a different way.
+    my $end_time    = $job_metrics->time;
+    my $delta_time  = $end_time - $start_time;
+    $job_metrics->add("ingest_unpack_seconds_total", $delta_time);
+    my $outdir_size = $job_metrics->dir_size($outdir);
+    $job_metrics->add("ingest_unpack_bytes_w_total", $outdir_size);
```

Comment:
unpack is also reading from something to unpack; maybe we should record that as well? That can be a future issue for sure.

Reply:
Yep, missed it. Added in [46341c8](https://github.com/hathitrust/feed/pull/128/commits/46341c80af90d93ad08ab9da5be4e71748c5e41b)